src/functions-and-arrays.js

```diff
@@ -52,14 +93,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let newArr = []
+  for (let i = 0; i < arr.length; i++){
+    if(!newArr.includes(arr[i])){
+      newArr += arr[i] + " "
+    }
+  }
+  return newArr
+}
+console.log(uniquifyArray(wordsUnique))
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, word) {
+  for(let i = 0; i < arr.length; i++){
```

Comment:
The logic is ok, but you are returning a string and the tests ask for Booleans (true or false). Also, you need to return 'null' when receiving an empty array.
Here's a solution you can use:

```
function doesWordExist(arr, word) {
  if(arr.length === 0) return null;
  for(let i = 0; i < arr.length; i++){
    if(arr.includes(word)){
      return true
    } else return false
  }
}
```